app/next-i18next.config.js

```diff
@@ -8,4 +8,8 @@ module.exports = {
     // See https://react.i18next.com/latest/trans-component#using-for-less-than-br-greater-than-and-other-simple-html-elements-in-translations-v-10-4-0
     transKeepBasicHtmlNodesFor: ['br', 'strong', 'i', 'p', 'em'],
   },
+  // Locale resources are loaded once when the server is started, which
+  // is good for production but not ideal for local development. Show
+  // updates to locale files without having to restart the server:
+  reloadOnPrerender: process.env.NODE_ENV === 'development',
```

Comment:
Notes for reviewers: Included from https://github.com/navapbc/template-application-nextjs/pull/56